testutils/src/main/java/com/hyperwallet/android/ui/testutils/util/DateConversion.java

```diff
@@ -0,0 +1,22 @@
+package com.hyperwallet.android.ui.testutils.util;
+
+
+import static android.text.format.DateUtils.formatDateTime;
+
+import android.content.Context;
+
+import com.hyperwallet.android.ui.common.util.DateUtils;
+
+import java.util.Date;
+
+public class DateConversion {
```

Comment:
This is in the wrong location, it should go into`hyperwallet-android-ui-sdk/testutils/src/main/java/com/hyperwallet/android/ui/testutils/util`.